lib/puppetserver/ca/config/puppet.rb

```diff
@@ -151,6 +155,9 @@ def resolve_settings(overrides = {})
           settings[:ca_ttl] = munge_ttl_setting(settings[:ca_ttl])
           settings[:certificate_revocation] = parse_crl_usage(settings[:certificate_revocation])
 
+          # rename dns_alt_names to subject_alt_names now that we support IP alt names
+          settings[:subject_alt_names] = overrides.fetch(:dns_alt_names, "puppet,$certname")
```

Comment:
I don't think it works because the name of the setting I'm pulling out of Puppet is different than the name I'm saving it under. But maybe I'm missing something?

Reply:
You are right.  I didnt even notice they were different